### DISCUSSION TITLE
Cannot completely delete .deb installation on Ubuntu Linux

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#4119] [Q&A]
I installed the .deb of ente auth on Ubuntu 24.10 but it will not completely uninstall from the system with:

sudo apt remove --purge ente

Does anyone know the proper command to completely remove the ente auth .deb installation?

**Comment by mrlinnth** [comment]
`sudo apt remove --purge enteauth`  should do the trick. 

Late answer but hope this helps for future query.
